todo.js

```diff
@@ -1 +1,140 @@
-// Write your code
+"use strict";
+
+const useState = React.useState;
+const useEffect = React.useEffect;
+
+const App = () => {
+  const [newText, setNewText] = useState("");
+  const [todos, setTodos] = useState(
+    JSON.parse(localStorage.getItem("todos")) || []
+  );
+  const [editText, setEditText] = useState("");
+
+  const handleCreate = () => {
+    if (!newText) {
+      return;
+    }
+    setNewText("");
+    const maxId = todos.length ? Math.max(...todos.map((todo) => todo.id)) : 0;
```

Comment:
JavaScript には falsy な値が多く、それを思い出す作業で読み手に負担がかかるので、条件を明示したほうが読みやすくなると思います。

```suggestion
    const maxId = todos.length === 0 ? Math.max(...todos.map((todo) => todo.id)) : 0;
```

Reply:
> JavaScript には falsy な値が多く、それを思い出す作業で読み手に負担がかかる

こちらの視点は欠けていましたので、以後認識しておきます🙏
（修正済）

Reply:
すみません、こちら条件を書き間違えていました...。元のプログラムと同じにするなら `todos.length !== 0` か `todos.length > 0` と書くべきでしたね。対応としては変更後のコードのままで大丈夫です。

Reply:
はい、条件が逆であることは気付けましたので大丈夫です🙆‍♂️